trio/_core/_multierror.py

```diff
@@ -381,11 +381,17 @@ def traceback_exception_init(
     limit=None,
     lookup_lines=True,
     capture_locals=False,
+    compact=False,
```

Comment:
Can we use *args, **kwargs here?

Reply:
I did that as first (as it is simpler), but figured it might be better from a static analysis perspective to add the actual new argument. I don't really care that much either way.